### ISSUE TITLE
helmfile lint does not add configured repositories

### ISSUE THREAD

**Issue by mhulscher** [roboll/helmfile#303]
Unlike `helmfile sync`, `helmfile lint` does not add missing repositories to local Helm. This will cause linting to fail unless the repository was previously configured.

```
$ helmfile --file helmfile.yaml lint
Error: repo incubator not found

err: exit status 1
```

`helmfile.yaml`
```
repositories:
- name: stable
  url: https://kubernetes-charts.storage.googleapis.com
- name: incubator
  url: https://kubernetes-charts-incubator.storage.googleapis.com

releases:
- name: vault
  namespace: vault
  chart: incubator/vault
  version: 0.12.0
```

**Comment by mhulscher** [comment]
This is not fixed in version v0.33.1

**Comment by mumoshu** [comment]
@mhulscher Would you mind sharing a full output of `helmfile --log-level debug lint`? (Beware not to expose repository passwords though!

**Comment by mhulscher** [comment]
This was indeed broken on v0.33.1 but seems to have been fixed in v0.33.2 👍

**Comment by mumoshu** [comment]
@mhulscher Yay! Thanks for your confirmation!
